cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_controller.go

```diff
@@ -342,7 +342,7 @@ func (t Template) ControllersPolicyEKS() *iamv1.PolicyDocument {
 			"iam:CreateServiceLinkedRole",
 		},
 		Resource: iamv1.Resources{
-			"arn:aws:iam::*:role/aws-service-role/eks-fargate-pods.amazonaws.com/AWSServiceRoleForAmazonEKSForFargate",
+			"arn:" + t.Spec.Partition + ":iam::*:role/aws-service-role/eks-fargate-pods.amazonaws.com/AWSServiceRoleForAmazonEKSForFargate",
```

Comment:
Hi. Could you please cover this change with a unit test? And please make sure that Partition defaults to `aws` so we don't get an empty string in there.

Reply:
Hi @Skarlso. About the UT, there is no existing test file to reference. The default was already populated in cmd/clusterawsadm/api/bootstrap/v1beta1/defaults.go as "aws".

Reply:
Ok 👍